cspreports/models.py

```diff
@@ -13,23 +15,21 @@
 )
 
 # Map of required CSP report fields to model fields
-REQUIRED_FIELD_MAP = (
-    ('document-uri', 'document_uri'),
-    ('referrer', 'referrer'),
-    ('blocked-uri', 'blocked_uri'),
-    ('violated-directive', 'violated_directive'),
-    ('original-policy', 'original_policy'),
+REQUIRED_FIELDS = (
+    ('document-uri'),
```

Comment:
I'd rather keep the `*_FIELD_MAP` definitions than `*_FIELDS`. Currently they differ only in dash/underscore, but that may not be case in future. Field maps allow us to map JSON fields to model fields arbitrarily, which may come handy someday.

To be clear I like the merge of integer and optional fields below.

Reply:
👍  done